about.md

```diff
@@ -10,7 +10,7 @@ security and privacy tools. Our dedicated team is at the helm of the OpenSSL
 project, ensuring secure digital interactions. Here, you'll meet the individuals
 who are collectively shaping a safer, privacy-respecting digital future.
 
-![team](/img/faces/openssl2023.png "Some of the OpenSSL team in 2023")
+![Some of the OpenSSL team in 2023](/img/faces/openssl2023.png "Some of the OpenSSL team in 2023")
```

Comment:
Maybe include a "listed below in alphabetic order".
I.e. to indicate that there is no relationship to the left to right in the photograph.

Reply:
Good point, done!